.github/workflows/test.yaml

```diff
@@ -260,7 +260,10 @@ jobs:
             #
           ]
         nox-shard: [
-            "trace",
+            "trace1",
```

Comment:
Can we add 'shard' to the name here to be more descriptive? like trace-shard-1 or trace_shard_1?